### PR TITLE
Unit test for RecurrentBookingValidator

### PR DESCRIPTION
**Pull request by hrybs**
LVJAVS-827 - http://ssu-jira.softserveinc.com/browse/LVJAVS-827

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -420,6 +422,20 @@
             <version>${org.mockito.mockito-all-version}</version>
         </dependency>
 
+            <dependency>
+                <groupId>org.powermock</groupId>
```
Comment (ValeryGutor) [inline pom.xml]: Why you use powermock?

Comment (hrybs) [inline pom.xml]: If we use only Mockito, then we can not test final class, static method and we have other constraints. So in this case I additionally used PowerMock, that doesn't have this restrictions

File: src/test/java/ua/softserveinc/tc/validator/RecurrentBookingValidatorTest.java [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    private RecurrentBookingValidatorImpl recurrentBookingValidator;
+
+    @Before
+    public void initValidBookingDto() {
+        when(bookingDto.getDaysOfWeek()).thenReturn("Mon Tue Fri");
+    }
+
+    @Test
+    public void testIsValidToInsertHasNullListOfBookingDto() {
+        testNullOrEmptyFailInsert("list of dro is null", null);
```
Comment (andryspo) [inline src/test/java/ua/softserveinc/tc/validator/RecurrentBookingValidatorTest.java]: dto not dro*

Comment (hrybs) [inline src/test/java/ua/softserveinc/tc/validator/RecurrentBookingValidatorTest.java]: Good attentiveness. I am going to correct it in other pull request.

**Pull request merged**